sympy/solvers/inequalities.py

```diff
@@ -539,7 +539,7 @@ def valid(x):
                 # expanded_e, expr and gen used from enclosing scope
                 v = expanded_e.subs(gen, x)
                 try:
-                    r = expr.func(v, 0)
+                    r = expand_mul(expr.func(v, 0))
```

Comment:
use `v = expanded_e.subs(gen, expand_mul(x))` above instead of using `expand_mul` here